plugins/kubeone.yaml

```diff
@@ -19,7 +19,10 @@ spec:
       uri: https://github.com/kubermatic/kubeone/releases/download/v1.3.0/kubeone_1.3.0_darwin_amd64.zip
       sha256: 3dc786156c48acfabb7f9ae39483ec2ea12983bb371d0f447095d5a0afeca96b
       bin: kubeone
-  shortDescription: Kubermatic KubeOne automate cluster operations on all your cloud, on-prem, edge, and IoT environments
+  shortDescription: Automate cluster operations on chosen environments
```

Comment:
Please be more specific what are "cluster operations". I also don't understand why you emphasisze "chosen environments". This seem obvious to me, so there's no need to mention it?